css/annotations/popup.scss

```diff
@@ -238,4 +240,23 @@ $button-hover-color: #e6ebf5;
 .highcharts-indicator-title {
   float: left;
   padding-bottom: 15px;
+}
+
+label[for="highcharts-input-search-indicators"] {
+  text-align: center;
+  font-weight: bold;
+  color: black;
+  margin-top: 0.5rem;
+}
+
+input[name="highcharts-input-search-indicators"] {
```

Comment:
I would replace it with the class, context selectors are not supported in the older IE.